src/Microsoft.VisualStudio.Editors/package/VBPackage.vb

```diff
@@ -26,7 +26,7 @@ Namespace Microsoft.VisualStudio.Editors
     '* In the future, we should consider moving to a RegPkg.exe model
     <
         Guid("67909B06-91E9-4F3E-AB50-495046BE9A9A"),
-        ProvideOptionPage(GetType(GeneralOptionPage), "Projects and Solutions", ".NET Core", 1400, 1400, True),
+        ProvideOptionPage(GetType(GeneralOptionPage), "Projects and Solutions", ".NET Core", 1400, 1400, True, 1600),
```

Comment:
This actually wasn't required because we have auto generating our .pkgdef turned off, but I added it just to clear any future confusion

Reply:
I really wish we could get that enabled like Roslyn does, it makes it easier - but we only have one option page so it isn't much of a priority

Reply:
@davkean do you know why we don't generate out pkgdef?

Reply:
Please sync with my changes and avoid changing this - this only adds confusion as its not used. This project hasn't been setup to auto-gen the pkgdef, there's a bug on it.